src/e-build.js

```diff
@@ -64,38 +64,41 @@ program
   .description('Build Electron and other targets.')
   .option('--list-targets', 'Show all supported targets', false)
   .option('--gen', 'Force a re-run of `gn gen` before building', false)
+  .option('-t|--target [target]', 'Forces a specific ninja target')
   .parse(process.argv);
 
-const pretty_targets = {
-  breakpad: 'third_party/breakpad:dump_sym',
-  chromedriver: 'electron:electron_chromedriver_zip',
-  electron: 'electron',
-  'electron:dist': 'electron:electron_dist_zip',
-  mksnapshot: 'electron:electron_mksnapshot_zip',
-  'node:headers': 'third_party/electron_node:headers',
-};
-
-if (program.listTargets) {
-  Object.keys(pretty_targets)
-    .sort()
-    .forEach(target => console.log(target));
-}
-
 try {
   const config = evmConfig.current();
+
+  const pretty_targets = {
+    breakpad: 'third_party/breakpad:dump_sym',
+    chromedriver: 'electron:electron_chromedriver_zip',
+    electron: 'electron',
+    'electron:dist': 'electron:electron_dist_zip',
+    mksnapshot: 'electron:electron_mksnapshot_zip',
+    'node:headers': 'third_party/electron_node:headers',
+    __default: config.defaultTarget || 'electron',
```

Comment:
This double-underscore prefix has me confused -- is this a user-facing key that one would type in on the command line (e.g. `e build __default`), or is it an internal placeholder for when one types `e build`?

If it's intended as a user-facing key, could we use `default` instead? The leading double underscore makes this look like an implementation variable rather than a user-facing option and feels weird.

If it's intended as an internal placeholder, dumping this internal key in the console.log on line 86 is a leaky impl. We could bypass this by removing the `__default` lines altogether and doing this:

```js
const pretty = Object.keys(pretty_targets).find(p => program.rawArgs.includes(p));
...
const target = pretty_targets[pretty] || config.defaultTarget || 'electron';
runNinja(config, target, args);
```

Either of these approaches (public key named `default` or removing `*default` altogether) would be fine; I don't have a preference

Reply:
It's now a public "default" and I also improved the behavior of `--list-targets`